conceptual/deployment/distributing.md

```diff
@@ -6,6 +6,8 @@ uid: distributing
 
 When your project uses aspects, you need to consider whether the projects that _reference_ your projects will also need to use aspects just because of this reference.
 
```

Comment:
I'm following the terminology used by Microsoft: https://docs.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files